internal/pkg/runner/job/job_runner.go

```diff
@@ -0,0 +1,86 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+//Package job provides support for invoking scheduled jobs
+
+package job
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/copilot-cli/internal/pkg/aws/cloudformation"
+	"github.com/aws/copilot-cli/internal/pkg/aws/stepfunctions"
+)
+
+type JobExecutor interface {
```

Comment:
Same here and the other interfaces in this package! We can keep this unexported if it's not referenced in other packages:
```suggestion
type jobExecutor interface {
```